### DISCUSSION TITLE
Humble release thread

### DISCUSSION THREAD

**Discussion by sea-bass** [moveit/moveit2#3131] [General]
As @rr-tom-noble suggested, we can use this thread to continue discussing Humble releases of MoveIt 2.

As of https://github.com/moveit/moveit2/issues/3012, a release 2.5.6 was cut.

However, there are already a few more backports/fixes on the Humble branch, and some more upcoming ones.

I would say as a first discussion, we should establish when all these desired changes are in good shape to kick off a 2.5.7. I'd roughly aim for end of 2024, so a month or so from now? Of course, depending on the actual changes needed and when that can be done.

**Comment by TSNoble** [comment]
Thanks for creating this thread, and end of December sounds good to me :+1:

**Comment by sea-bass** [comment]
BTW (@TSNoble), seems like MoveIt 2.5.6 made it in for the Humble sync that will be released in a few days: https://discourse.ros.org/t/preparing-for-humble-sync-and-patch-release-2024-12-02/40922

**Comment by sea-bass** [comment]
Since https://github.com/moveit/moveit2/pull/1821 fixes a bug, I'm inclined for us to get 2.5.7 in for the next Humble sync.

@TSNoble (or anyone else), do you have any more Humble PRs coming in in the next few days? Probably aiming to have the new release before ~Christmas (Dec 25).

**Reply by TSNoble** [reply]
None from me (planning to finish off my unit test PR, but on Christmas leave at the moment) :slightly_smiling_face:

**Reply by sea-bass** [reply]
Ah, looks like I just missed it. There was another Humble sync on December 20. Will catch the next one in the new year.

**Reply by sea-bass** [reply]
2.5.7 is queued up in `rosdistro`: https://github.com/ros/rosdistro/pull/44074

**Comment by sea-bass** [comment]
I think it's time for 2.5.8 to happen soon -- see https://github.com/orgs/moveit/discussions/3310#discussioncomment-12061924 for more details.

**Reply by sea-bass** [reply]
2.5.8 is queued up in `rosdistro` : https://github.com/ros/rosdistro/pull/44410

**Comment by riv-tnoble** [comment]
@sea-bass Thank you for keeping this alive! Although I've been less active recently, it's been wonderful being able to receive more frequent updates :slightly_smiling_face:

**Comment by sea-bass** [comment]
I just had to release Rolling and Jazzy for legitimate reasons, and simply didn't want to leave old Humble out.

2.5.9 is queued up: https://github.com/ros/rosdistro/pull/45196
